src/vcl/ctx.rs

```diff
@@ -127,13 +127,13 @@ impl<'a> Ctx<'a> {
     }
 
     pub fn cached_req_body(&mut self) -> Result<Vec<&'a [u8]>, crate::vcl::Error> {
-        unsafe extern "C" fn chunk_collector<'a>(
+        unsafe extern "C" fn chunk_collector(
```

Comment:
Check if this change is correct

Reply:
if it compiles, I trust rust :-D

but yeah, it looks like it can be elided